Makefile

```diff
@@ -210,3 +219,45 @@ push-image: build-image
 	docker tag "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)" "$(OPERATOR_IMAGE):latest"
 	docker push "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)"
 	docker push "$(OPERATOR_IMAGE):latest"
+
+.PHONY: local
+## Run operator locally
+local: deploy-clean deploy-rbac deploy-crds deploy-cr
+	$(Q)operator-sdk up local
+
+.PHONY: deploy-rbac
+## Setup service account and deploy RBAC
+deploy-rbac:
+	$(Q)kubectl create -f deploy/service_account.yaml
+	$(Q)kubectl create -f deploy/role.yaml
+	$(Q)kubectl create -f deploy/role_binding.yaml
+
+.PHONY: deploy-crds
+## Deploy CRD
+deploy-crds:
+	$(Q)kubectl create -f deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml
+
+.PHONY: deploy-cr
+## Deploy CRs
+deploy-cr:
+	$(Q)kubectl apply -f deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
+
+.PHONY: deploy-clean
+## Removing CRDs and CRs
+deploy-clean:
+	$(Q)-kubectl delete -f deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
+	$(Q)-kubectl delete -f deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml
+	$(Q)-kubectl delete -f deploy/operator.yaml
+	$(Q)-kubectl delete -f deploy/role_binding.yaml
+	$(Q)-kubectl delete -f deploy/role.yaml
+	$(Q)-kubectl delete -f deploy/service_account.yaml
+
+
+#---------------------------------------------------------
+# Cleaning targets
+#---------------------------------------------------------
+
+.PHONY: clean
+## Removes temp directories
+clean:
+	$(Q)-rm -rf ${V_FLAG} ./out
```

Comment:
Do we really need to silence the command itself here?
The hyphen here (`$(Q)-`) makes the command invisible, right?

Reply:
Its not necessary it was just a precaution, if due to some reason it gives any error then it should halt any execution(if clean target is added as dependency to some other target in future).